.github/workflows/build-maven.yml

```diff
@@ -32,4 +32,4 @@ jobs:
           ${{ runner.os }}-maven-
 
     - name: Run Maven
-      run: mvn -B clean verify com.mycila:license-maven-plugin:check
+      run: mvn -B clean -P !autoFormat,checkFormat verify
```

Comment:
🔧 With `activeByDefault` you don't need to disable the `autoFormat` profile. Simply by activating the `checkFormat` profile, the `autoFormat` profile will already be disabled.